en/companion_computer/video_streaming.md

```diff
@@ -1,33 +1,40 @@
 # Video Streaming (Companion Computer/QGroundControl)
 
-PX4-based vehicles support video streaming using a camera connected to a companion computer.
-GStreamer is used to send the video to *QGroundControl* over an IP link.
+PX4-based vehicles support video streaming using a camera connected to a [companion computer](../companion_computer/README.md).
 
 :::note
 You can't video stream directly from a camera connected to PX4.
 :::
 
-To support streaming use cases you will need to install *GStreamer* development packages on both your companion computer and on the system running *QGroundControl*.
-
-*QGroundControl* uses GStreamer 1.14.4 and a stripped down version of *QtGstreamer* to support UDP RTP and RSTP video streaming.
+GStreamer is used to send the video to _QGroundControl_ over an IP link.
+To support streaming use cases you will need to install _GStreamer_ development packages on both your companion computer and on the system running _QGroundControl_.
+_QGroundControl_ uses GStreamer 1.14.4 and a stripped down version of _QtGstreamer_ to support UDP RTP and RSTP video streaming.
 
 ## Companion Computer Setup
 
-General instructions for installing *GStreamer* and starting the stream on your companion computer are provided in the [QGroundControl VideoReceiver README](https://github.com/mavlink/qgroundcontrol/blob/master/src/VideoReceiver/README.md).
+General instructions for installing _GStreamer_ and starting the stream on a companion computer are provided in the [QGroundControl VideoReceiver README](https://github.com/mavlink/qgroundcontrol/blob/master/src/VideoReceiver/README.md).
+
+The setup of cameras and data links depends on many factors.
+Examples in this library are listed below (note, these are options, not "recommended"):
+
+- [Video Streaming using WFB-ng Wifi](../companion_computer/video_streaming_wfb_ng_wifi.md) (Tutorial): Using RaPi and WiFi module in unconnected (broadcast) mode to stream video and as a bidirectional telemetry link.
```

Comment:
So I've added this link from companion computers to the tutorial. It would be good if we had more setup options/instructions such as a high power microhard radio or LTS. We don't.